stuff/cron/test_plagiarism_detector.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from typing import List
+import pytest
+import argparse
+import calendar
+import collections
+import datetime
+import json
+import logging
+import operator
+import sys
+import os
+sys.path.insert(
+    0,
+    os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+                 "."))
+import lib.db  # pylint: disable=wrong-import-position
+import lib.logs  # pylint: disable=wrong-import-position
+ 
+from plagiarism_detector import get_contests # can only import if in the same directory.
+
+def main() -> None:
+
+    parser = argparse.ArgumentParser(
+        description='Runs the Plagiarism Detector')
+    parser.add_argument('--local-downloader-dir')
+
+    lib.db.configure_parser(parser)
+    lib.logs.configure_parser(parser)
+
+    args = parser.parse_args()
+    args.verbose = True
+    lib.logs.init(parser.prog, args)
+
+    logging.info('started')
+    dbconn = lib.db.connect(lib.db.DatabaseConnectionArguments.from_args(args))
```

Comment:
```suggestion
    dbconn = lib.db.connect(lib.db.DatabaseConnectionArguments(
      host='localhost',
      port=...
      ...
    ))
```

in the test we should build the arguments directly instead of from arguments (that shouldn't be used in tests).

Reply:
Done.